### PR TITLE
Pancancer 833

### PR DESCRIPTION
**Pull request by lindaxiang**

### CODE REVIEW

File: pcawg_metadata_parser/generate_release.py [hunk 1]
```diff
@@ -316,16 +401,18 @@ def generate_variant_calling_info(pilot_tsv, variant_calling, vcf):
         pilot_tsv[get_formal_vcf_name(v)+'_repo'] = []
         pilot_tsv[get_formal_vcf_name(v)+'_gnos_id'] = []
         pilot_tsv[get_formal_vcf_name(v)+'_file_name_prefix'] = []
+        pilot_tsv['is_aug2015_'+get_formal_vcf_name(v)] = []
         for specimen in variant_calling:
             if specimen.get(get_formal_vcf_name(v)):
                 pilot_tsv[get_formal_vcf_name(v)+'_repo'].append(specimen.get(get_formal_vcf_name(v)).get('gnos_repo'))
```
Comment (junjun-zhang) [inline pcawg_metadata_parser/generate_release.py]: This is reported by Christina, for multiple tumor specimens, all of the calls are in one singe GNOS, so _gnos_repo_ should be a string (not a list of strings), it is the same as _gnos_id_

File: pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+3c78eef3-d385-4c9f-81cd-d2948ef00f05
+ffe3eee8-2b5e-4eb2-b42e-7d9d2e4b2203
+a19ba8e0-8ca0-40b8-b40e-92446d8d347b
+f3542b74-965a-48d7-a707-3b0a04f173ac
+c4cd1ffc-6f9a-4c6f-98ad-55997a73245d
```
Comment (junjun-zhang) [inline pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt]: what is this file: pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt?

File: pcawg_metadata_parser/generate_release.py [hunk 3]
```diff
@@ -37,34 +37,91 @@
...
+                                      },
+                                      {
+                                         "term":{
+                                            "flags.is_embl_variant_calling_performed":[
+                                               "F"
+                                            ]
+                                         }
+                                      } 
+                                ]
+                           }
```
Comment (junjun-zhang) [inline pcawg_metadata_parser/generate_release.py:86]: These filtering criteria (the above and other places) ideally are optional and can be specific by command line arguments. The reason is that we will use this script to generate different kind of summary reports/lists (other than just a specific PCAWG data release, which does need more stringent filtering), for a general overview report, we'd like to have minimum filtering.

Maybe we merge for now, and add this feature later.

File: pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt (hunk 2 above)
Comment (lindaxiang) [inline pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt]: These should be the excluded gnos ids when we generated oct2015 release. They may not be effective in future as we have all the gnos_ids in oct2015 release.

-Linda

On Nov 20, 2015, at 3:50 PM, Junjun Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:

In pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txthttps://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/43#discussion_r45516819:

> @@ -0,0 +1,5 @@
> +3c78eef3-d385-4c9f-81cd-d2948ef00f05
> +ffe3eee8-2b5e-4eb2-b42e-7d9d2e4b2203
> +a19ba8e0-8ca0-40b8-b40e-92446d8d347b
> +f3542b74-965a-48d7-a707-3b0a04f173ac
> +c4cd1ffc-6f9a-4c6f-98ad-55997a73245d

what is this file: pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt?

Comment (junjun-zhang) [inline pcawg_metadata_parser/release_oct2015/excluded_gnos_ids.txt]: ok, in that case, please remove the file for now. if we need something in the future, we can add. thanks!

File: pcawg_metadata_parser/generate_release.py (hunk 3 above)
Comment (lindaxiang) [inline pcawg_metadata_parser/generate_release.py:86]: Yes. Agree.

On Nov 20, 2015, at 3:57 PM, Junjun Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:

In pcawg_metadata_parser/generate_release.pyhttps://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/43#discussion_r45517547:

> -                                      {
> -                                         "term":{
> -                                            "flags.ishttp://flags.is_dkfz_variant_calling_performed":[
> -                                               "F"
> -                                            ]
> -                                         }
> -                                      },
> -                                      {
> -                                         "term":{
> -                                            "flags.ishttp://flags.is_embl_variant_calling_performed":[
> -                                               "F"
> -                                            ]
> -                                         }
> -                                      }
> -                                ]
> -                           }

These filtering criteria ideally are optional and can be specific by command line arguments. The reason is that we will use this script to generate different kind of summary reports/lists (other than just a specific PCAWG data release, which does need more stringent filtering), for a general overview report, we'd like to have minimum filtering.

Maybe we merge for now, and add this feature later.

**Pull request merged**